Libs/MRML/Widgets/qMRMLPlotWidget.cxx

```diff
@@ -116,37 +119,47 @@ void qMRMLPlotWidget::setMRMLPlotViewNode(vtkMRMLPlotViewNode* newPlotViewNode)
   d->PlotController->setMRMLPlotViewNode(newPlotViewNode);
 }
 
-// --------------------------------------------------------------------------
+//--------------------------------------------------------------------------
+void qMRMLPlotWidget::setMRMLAbstractViewNode(vtkMRMLAbstractViewNode* newPlotViewNode)
+{
+  this->setMRMLPlotViewNode(vtkMRMLPlotViewNode::SafeDownCast(newPlotViewNode));
```

Comment:
we should log a warning if a wrong view node type is provided (not just here but in all other widget classes, too).